XCTWebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -58,6 +58,17 @@ + (NSArray *)routes
         XCUIElement *element = [elementCache elementForIndex:[request.parameters[@"id"] integerValue]];
         return FBResponseDictionaryWithStatus(FBCommandStatusNoError, element.wdLocation);
     }],
+    [[FBRoute GET:@"/element/:id/location_in_view"] respond:^ id<FBResponsePayload> (FBRouteRequest *request) {
+        FBXCTElementCache *elementCache = (FBXCTElementCache *)request.session.elementCache;
+        XCUIElement *element = [elementCache elementForIndex:[request.parameters[@"id"] integerValue]];
```

Comment:
Could you refactor code a bit and reuse `+ (id<FBResponsePayload>)handleScrollElementToVisible: withRequest:` instead?
Eg. You could add method that returns `BOOL` and use it inside `+ (id<FBResponsePayload>)handleScrollElementToVisible: withRequest:` an in your method.